frontend/common/src/lang/fr.json

```diff
@@ -1353,5 +1353,9 @@
   "VOheZB": {
     "defaultMessage": "<primary>{role}</primary> à {organization}",
     "description": "Role at organization in work experience block of skill accordion."
+  },
+  "DGR0FU": {
+    "defaultMessage": "Modifier la taille de la page",
```

Comment:
```suggestion
    "defaultMessage": "Taille de la page",
```

Reply:
@esizer you'll need to re-generate as `DGR0FU` will no longer be the id since the original English value has changed as well.